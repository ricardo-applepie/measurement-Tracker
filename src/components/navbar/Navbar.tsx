import React from "react";

// Boostrap navbar  .
function Navbar() {
  return (
    <div>
      <div className="pos-f-t">
        <div className="collapse" id="navbarToggleExternalContent">
          <div className="bg-dark p-4">
            <h4 className="text-white">Home</h4>
          </div>
        </div>
        <nav className="navbar navbar-dark bg-dark">
          <button
            className="navbar-toggler"
            type="button"
            data-toggle="collapse"
            data-target="#navbarToggleExternalContent"
            aria-controls="navbarToggleExternalContent"
            aria-expanded="false"
            aria-label="Toggle navigation"
          >
            <span className="navbar-toggler-icon"></span>
          </button>
        </nav>
      </div>
    </div>
  );
}

export default Navbar;
