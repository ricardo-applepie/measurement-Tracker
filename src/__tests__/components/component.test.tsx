import React from "react";
import renderer from "react-test-renderer";
import SimpleButton from "../../components/button/Button";
import Navbar from "../../components/navbar/Navbar";

test("testing if button is rendered ", () => {
  const component = renderer.create(<SimpleButton buttonText="btnText" />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // manually trigger the callback
});

test("testing if navbar is rendered ", () => {
  const component = renderer.create(<Navbar />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // manually trigger the callback
});
export default {};
