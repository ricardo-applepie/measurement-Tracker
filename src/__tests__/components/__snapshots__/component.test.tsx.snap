// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing if Icon is rendered  1`] = `
<div>
  <svg
    data-testid="DeleteIcon"
  />
</div>
`;

exports[`testing if button is rendered  1`] = `
<div>
  <button
    className="button"
    color="primary"
  >
    btnText
  </button>
</div>
`;

exports[`testing if navbar is rendered  1`] = `
<div>
  <div
    className="pos-f-t"
  >
    <div
      className="collapse"
      id="navbarToggleExternalContent"
    >
      <div
        className="bg-dark p-4"
      >
        <h4
          className="text-white"
        >
          Home
        </h4>
      </div>
    </div>
    <nav
      className="navbar navbar-dark bg-dark"
    >
      <button
        aria-controls="navbarToggleExternalContent"
        aria-expanded="false"
        aria-label="Toggle navigation"
        className="navbar-toggler"
        data-target="#navbarToggleExternalContent"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
    </nav>
  </div>
</div>
`;
